azext_edge/tests/edge/support/create_bundle/test_orc_int.py

```diff
@@ -0,0 +1,62 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from azext_edge.edge.common import OpsServiceType
+from .helpers import check_non_custom_file_objs, get_file_map, run_bundle_command
+
+logger = get_logger(__name__)
+
+
+def test_create_bundle_orc(init_setup, tracked_files):
+    """Test for ensuring file names and content. ONLY CHECKS orc."""
+    ops_service = OpsServiceType.orc.value
+    command = f"az iot ops support create-bundle --ops-service {ops_service}"
+    walk_result = run_bundle_command(command=command, tracked_files=tracked_files)
+    file_map = get_file_map(walk_result, ops_service)
+
+    # TODO: add in expected for each
+    # orc
+    # do we always have these? What cases do they have them vs not?
+    # how can names change?
+    for config in file_map.get("instance", []):
```

Comment:
These assertions are like, if the CR is there, assert its version, but as soon as an API version is incremented these tests break. Also there is a gap on asserting an expected CR is not available in the support bundle. We can fetch these resources from the cluster and dynamically assert KPIs like version so it's more resilient.